go/libraries/utils/argparser/parser.go

```diff
@@ -235,7 +235,7 @@ func (ap *ArgParser) Parse(args []string) (*ArgParseResults, error) {
 
 		for _, opt := range modalOpts {
 			if _, exists := results[opt.Name]; exists {
-				return nil, errors.New("error: multiple values provided for `" + opt.Name + "'")
```

Comment:
These quotes are correct as it. They follow shell style, not MySQL backtick-escaping